index.html

```diff
@@ -674,12 +677,12 @@ <h3>Validate Claim</h3>
           is the domain of validation.
           <br><br>
           It's important to note that the bounds of authority for a given
-          issuer are specific claim types made by them and relied upon by
+          issuer are specific credential types made by them and relied upon by
           others. While a region's DMV (Department of Motor Vehicles)
           or equivalent agency might be an acceptable authority for an
           individual's driving privilege, it should probably not be treated
           as authoritative for an individual's hair color or current
-          address, even though DMVs regularly include such claims in
+          address, even though such information is regularly included in
```

Comment:
```suggestion
          address, even though such claims are often included in
```
```suggestion
          address, even though such information is regularly included in
```

Ditto for the previous comment. In this case, it is the non-authoritative claims that are being considered. Reducing to "information" ignores the claim-based vetting of sources as described previously.